_episodes/08-putting-it-all-together.md

```diff
@@ -86,32 +89,37 @@ also change the name of the columns in the DataFrame like this:
 ~~~
 {: .language-python}
 
-## Make a line plot of your data
+## Matplotlib plot library
 
-Matplotlib is a Python library that can be used to visualize data. The
-toolbox `matplotlib.pyplot` is a collection of functions that make matplotlib
-work like MATLAB. In most cases, this is all that you will need to use, but
-there are many other useful tools in matplotlib that you should explore.
+[Matplotlib](http://matplotlib.org/) is a Python package used widely throughout the scientific Python community to produce high quality and publication-ready graphics. It supports a wide range of output formats including PNG (and other raster formats), PostScript/EPS, PDF and SVG. 
 
-We will cover a few basic commands for formatting plots in this lesson. A great
-resource for help styling your figures is the matplotlib gallery
+At the same time, matplotlib is the actual engine behind the plotting capabilities of both the Pandas as well as the plotnine package. So, if you you call the `.plot` functionality of Pandas, as we did in the previous episodes, you actually used the matplotlib package:
```

Comment:
`as well as` ~> `and` (this is my personal preference, so you can ignore it if you disagree)

Reply:
Fix: `So, if you call....` and `you actually used` (either: `When you called` or `you actually use`)